assets/chezmoi.io/docs/reference/templates/passhole-functions/passhole.md

```diff
@@ -0,0 +1,6 @@
+# passhole(ph) *command*
+`ph` returns structured data from a [Keepass](https://keepass.info/) database. Users can add, modify, delete, search entries in the database by using ph's commands: "list,ls,add,remove,rm,move,mv,show,edit,type,init,grep,dump,info,kill"
```

Comment:
It looks like this text was coped from the output of `ph -h`. The list of ph's commands is not relevant to chezmoi users, only `show` is used by chezmoi.